src/components/dialogs/create-contingency-list-dialog.js

```diff
@@ -40,6 +49,8 @@ export const CreateContingencyListDialog = ({ open, onClose }) => {
     const [contingencyListType, setContingencyListType] = React.useState(
         ContingencyListType.SCRIPT
     );
+    const [choosedContingencyListType, setChoosedContingencyListType] =
```

Comment:
chosen

Reply:
Do I change it in create-filter-dialog.js too ? I lifted it from there.

Reply:
Fixed in the other file in another PR.

Reply:
Fixed.